File: ITManagementClient/Models/RequestModels/Projects/CreateProjectRequestModel.cs
using System;

namespace ITManagementClient.Models.RequestModels.Projects
{
    public class CreateProjectRequestModel
    {
        public string Title { get; set; }

        public string Description { get; set; }

        public string TechnologiesStack { get; set; }

        public DateTime StartDate { get; set; }
    }
}

File: ITManagementServer/src/main/java/models/requestModels/projects/CreateProjectRequestModel.java
package models.requestModels.projects;

import java.sql.Date;

public class CreateProjectRequestModel {
    private String Title;
    private String Description;
    private String TechnologiesStack;
    private Date StartDate;


    public String getTitle() {
        return Title;
    }

    public void setTitle(String title) {
        Title = title;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String description) {
        Description = description;
    }

    public String getTechnologiesStack() {
        return TechnologiesStack;
    }

    public void setTechnologiesStack(String technologiesStack) {
        TechnologiesStack = technologiesStack;
    }

    public Date getStartDate() {
        return StartDate;
    }

    public void setStartDate(Date startDate) {
        StartDate = startDate;
    }
}